packages/shared/src/components/squads/SquadTabs.tsx

```diff
@@ -25,10 +27,10 @@ export function SquadTabs({
       : SquadTab.PendingPosts;
 
     return {
-      [SquadTab.Settings]: `${host}/edit`,
+      ...(showSettings && { [SquadTab.Settings]: `${host}/edit` }),
       [title]: `${host}/moderate`,
     };
-  }, [handle, pendingCount]);
+  }, [handle, pendingCount, showSettings]);
```

Comment:
Before this users can see the `Settings` tab, now we check outside if user is moderator and pass `showSettings`.